fenix/app/src/test/java/org/mozilla/fenix/onboarding/DefaultOnboardingControllerTest.kt

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.slot
+import io.mockk.unmockkObject
+import io.mockk.verify
+import org.junit.Assert
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mozilla.fenix.HomeActivity
+import org.mozilla.fenix.onboarding.controller.DefaultOnboardingController
+import org.mozilla.fenix.settings.SupportUtils
+
+class DefaultOnboardingControllerTest {
+
+    private val activity: HomeActivity = mockk(relaxed = true)
+
+    @Test
+    fun handleStartBrowsingClicked() {
+        var hideOnboardingInvoked = false
+        createController(hideOnboarding = { hideOnboardingInvoked = true }).handleStartBrowsingClicked()
+
+        Assert.assertTrue(hideOnboardingInvoked)
```

Comment:
did you mean to make this `Assert.assertTrue(...)` instead of just `assertTrue(...)`?

Reply:
Yes.